tests/testthat/test-merge_sce_list.R

```diff
@@ -417,39 +498,61 @@ test_that("merging SCEs with different altExp features works as expected, with a
     preserve_rowdata_cols = c("gene_names")
   )
 
-  merged_altexp <- altExp(merged_sce)
+  expect_true(altExpNames(merged_sce) == "adt")
 
+})
 
-  expect_true(altExpNames(merged_sce) == altexp_name)
-  expect_equal(dim(merged_altexp), c(num_altexp_features, total_cells))
-  expect_equal(rownames(merged_altexp), full_altexp_features)
 
-  expected_colnames <- sce_list_with_altexp |>
-    purrr::imap(
-      \(sce, sce_name) glue::glue("{sce_name}-{colnames(sce)}")
-    ) |>
-    unlist() |>
-    unname()
-  expect_equal(colnames(merged_altexp), expected_colnames)
+## Other tests ------------------
+
+test_that("build_na_matrix works as expected",{
+
+  rows <- letters[1:5]
+  cols  <- letters[6:10]
+  sparse_mat <- build_na_matrix(
+    "name",
+    rows,
+    cols
+  )
+
+  expect_true(
+    class(sparse_mat) == "lgeMatrix"
+  )
+
+  expect_equal(
+    rownames(sparse_mat), rows
+  )
+
+  expect_equal(
+    colnames(sparse_mat), cols
+  )
 
 })
 
 
-test_that("merging SCEs with altExps works as expected when include_altexps = FALSE", {
+test_that("check_altexps passes when it should pass", {
```

Comment:
Just noting that these should change with the function name change that I would like to see.